PxWeb/Controllers/Api2/NavigationApiController.cs

```diff
@@ -18,38 +18,28 @@
 using PxWeb.Helper.Api2;
 using PCAxis.Menu;
 using PxWeb.Mappers;
+using PxWeb.Api2.Server.Models;
 
 namespace PxWeb.Controllers.Api2
 {
     /// <summary>
     /// 
     /// </summary>
     [ApiController]
-    public class NavigationApiController : ControllerBase
+    public class NavigationApiController : PxWeb.Api2.Server.Controllers.NavigationApiController
     {
         private readonly IDataSource _dataSource;
         private readonly ILanguageHelper _languageHelper;
-        private readonly IResponseMapper _responseMapper;   
-        
-        public NavigationApiController(IDataSource dataSource, ILanguageHelper languageHelper, IResponseMapper responseMapper )
+        private readonly IResponseMapper _responseMapper;
+
+        public NavigationApiController(IDataSource dataSource, ILanguageHelper languageHelper, IResponseMapper responseMapper)
         {
             _dataSource = dataSource;
             _languageHelper = languageHelper;
-            _responseMapper = responseMapper;   
+            _responseMapper = responseMapper;
         }
 
-        /// <summary>
```

Comment:
Add updated method comments to the controllers

Reply:
Comments are now added to controller endpoints